LayoutTests/http/tests/paymentrequest/paymentrequest-features.https.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<script src="/js-test-resources/ui-helper.js"></script>
+<script src="/resources/js-test-pre.js"></script>
```

Comment:
Nit: New tests don't need `js-test-pre.js` and `js-test-post.js`, can just use `js-test.js`.